README.md

```diff
@@ -14,25 +14,168 @@ More information can be found in our [Wiki Pages](https://github.com/hydroshare/
+    Some WARNINGs are normal. 
+
+    HydroShare is available in your browser at http://localhost or http://localhost:8080  in case you are running inside a VM
+
+    The default admin page is http://localhost/admin
+
+    The default admin account is admin:default
+
+    Swagger API docs http://localhost/hsapi/
+
+6. Start & Stop & Log
+
+To start HydroShare, only need to open a windows shell, change to HydroShare code directory then run
+
+docker-compose -f local-dev.yml (up | down) [-d] [--build]
+
+Note bracketed -d for daemon is optional and you don’t paste in the brackets
+
+    Use -d option in case you want to type new command on this windows or don’t want to see real-time output log.
+
+    Use --build option in case docker keeps image in cache and does not update correctly while modifying the Dockerfile and working with PyCharm
+
+CREATE NEW ACCOUNT - This is the same as it's always been in HydroShare. Ask a teammate or hack at it. Basically open a hydroshare console window then use the UI to sign up for a new account and watch the hydroshare container console (docker logs hydroshare) for a verification link and paste that into your browser and save the new account in the UI.
+
+If your develop machine is slow, the defaultworker container may not work properly because it won’t see iRODS containers alive after timeout. Please wait seconds, open new terminal window then restart this container by the command: docker  restart  defaultworker, or just only start the whole HydroShare system by this command: ./slow-start.sh (no need to run docker-compose  -f  local-dev.yml  up) but by this way, you will not have native log window.
+
+
+
+To stop HydroShare, only need to close the running windows or open a new windows then run
+
+docker-compose -f local-dev.yml down
+
+All data is persisted for the next start.
+
+
+
+To see the logs in case you start with -d option, open a windows then run
+
+docker-compose -f local-dev.yml logs
+
+Or
+
+docker logs <container name>
+
+
+
+Warning Suppression
+
+If you are working on communities, it’s recommended you uncomment lines 197, 198, 207 in local_settings.py (Timezone stuff and silencing Mezz warnings). The Communities Developer Doc is here: https://docs.google.com/document/d/12VXZhqpMZCkaci5QviGqfg_xNZhk1fy-Zl-rFdlR0P8
+Sidenote on working Quickly
+
+Only docker-compose down when you need a deeper change (or maybe never) or are troubleshooting something. This keeps things very fast, especially if you use the PyCharm play/run and debug buttons.
+Python Unit Tests
+
+Some tests may fail this is work in progress as the build evolves...
+
+ 
+
+# when containers are already running in another terminal/shell
+
+docker exec -it hydroshare python manage.py test -v 3 --failfast
+
+
+
+Branching
+When you activate a new branch, just bring the stack down and up again. Sometimes you can get away with a warm restart of the stack or even relying on the Django debug mode (doing nothing but waiting). 
 
 ## Usage
 
```

Comment:
we also have the section about setting up debugging that seems to be missing and should be included, I'm open to adding it here to keep it all together or leaving it as a separate doc:

Reply:
Changed the URL to point to google doc vs github.